src/chains/views.py

```diff
@@ -10,7 +10,7 @@ class ChainsListView(ListAPIView):
     pagination_class = LimitOffsetPagination
     pagination_class.max_limit = 10
     pagination_class.default_limit = 10
-    queryset = Chain.objects.all()
+    queryset = Chain.objects.order_by("relevance", "id")
```

Comment:
Don't do the sorting here: https://www.django-rest-framework.org/api-guide/filtering/#specifying-a-default-ordering

Reply:
And maybe allow the users to order by name and relevance if they want

Reply:
Oh I saw this but since it added additional optional parameters for the clients to sort I decided to be more conservative. I can add this 👍

Reply:
Changed in 80ed735